sale_company_currency/views/sale_order_view.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <!-- SO -->
+    <record id="view_order_tree" model="ir.ui.view">
+      <field name="name">sale.order.tree</field>
+      <field name="model">sale.order</field>
+      <field name="inherit_id" ref="sale.view_order_tree"/>
+      <field name="arch" type="xml">
+
+        <!-- Remove multicurrencies sums (makes no sense)-->
```

Comment:
@elicoidal Having this sum on the bottom can be usefull but it is not working properly out of the box. If you have SO in EUR and USD, it will sums them up without taking in account the currency rate.
The idea of the filtering is good, but unless you activate it will output a wrong number and that's exactly what we try to avoid here.